SwerveControllerCommand/robotcontainer.py

```diff
@@ -0,0 +1,109 @@
+        """
+        self.robotDrive = DriveSubsystem()
+
+        # Driver Controller
+        self.driverController = XboxController(
+            constants.OIConstants.kDriverControllerPort
+        )
+
+        """
+        Configure default commands
+        """
+        self.robotDrive.setDefaultCommand(
+            # The left stick controls translation of the robot.
+            # turning is controlled by the X axis of the right stick.
+            commands2.RunCommand(
+                lambda: self.robotDrive.drive(
+                    # Multiply by max speed to map the joystick unitless inputs to actual units.
+                    # This will map the [-1, 1] to [max speed backwards, max speed forwards],
+                    # converting them to actual units.
+                    self.driverController.getYChannel()
+                    * constants.DriveConstants.kMaxSpeedMetersPerSecond,
+                    self.driverController.getXChannel()
+                    * constants.DriveConstants.kMaxSpeedMetersPerSecond,
+                    self.driverController.getX(constants.Hand.kRight)
+                    * constants.ModuleConstants.kMaxModuleAngularSpeedRadiansPerSecond,
+                    False,
+                ),
+                self.robotDrive,
+            )
+        )
+
+    def configureButtonBindings(self):
+        pass
+
+    def getAutonomousCommand(self):
+        """
+        Create config for trajectory
+        """
+        config = TrajectoryConfig(
+            constants.AutoConstants.kMaxSpeedMetersPerSecond,
+            constants.AutoConstants.kMaxAccelerationMetersPerSecondSquared,
+        ).setKinematics(constants.DriveConstants.kDriveKinematics)
+
+        # An example trajectory to follow. All units in meters.
+
+        example_trajectory = TrajectoryGenerator.generateTrajectory(
+            # Start at the origin facing the +X direction
+            Pose2d(0, 0, Rotation2d(0)),
+            # Pass through these two interior waypoints, making an 's' curve path
+            [Translation2d(1, 1), Translation2d(2, -1)],
+            # End 3 meters straight ahead of where we started, facing forward
+            Pose2d(3, 0, Rotation2d(0)),
+            config,
+        )
+
+        theta_controller = ProfiledPIDController(
+            constants.AutoConstants.kPThetaController,
+            0,
+            0,
+            constants.AutoConstants.kThetaControllerConstraints,
+        )
+        theta_controller.enableContinuousInput(-math.pi, math.pi)
+
+        swerve_controller_command = commands2.Swerve4ControllerCommand(
+            example_trajectory,
+            self.robotDrive.getPose,
+            # Functional interface to feed supplier
+            constants.DriveConstants.kDriveKinematics,
+            # Position controllers
+            PIDController(constants.AutoConstants.kPXController, 0, 0),
+            PIDController(constants.AutoConstants.kPYController, 0, 0),
+            theta_controller,
```

Comment:
This will need to be adjusted to match https://github.com/robotpy/robotpy-commands-v2/pull/45.